dev-java/netty-tcnative/netty-tcnative-1.1.33.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="5"
+
+WANT_AUTOCONF="latest"
+WANT_AUTOMAKE="latest"
+
+inherit autotools autotools-utils eutils java-pkg-2 java-pkg-simple
+
+DESCRIPTION="Tomcat Native fork for Netty"
+
+SLOT="0"
+SRC_URI="https://github.com/netty/${PN}/archive/${PV}.zip"
```

Comment:
You need to rename the distfile, something like `-> ${P}.zip`. Putting `1.1.33.zip` in distdir is never a good idea.